### PR TITLE
Scapegoat remove warnings 4ci

### PR DESCRIPTION
**Pull request by carloscaldas**
Closes https://github.com/LBNL-UCB-STI/beam/issues/2988
Depends on https://github.com/LBNL-UCB-STI/beam/pull/2983

**Removed 35 warnings**

This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/2989)

### CODE REVIEW

File: src/main/scala/beam/agentsim/agents/modalbehaviors/ChoosesMode.scala [hunk 1]
```diff
@@ -38,7 +38,7 @@ import beam.router.RoutingWorker
   */
 trait ChoosesMode {
   this: PersonAgent => // Self type restricts this trait to only mix into a PersonAgent
-  private implicit val executionContext: ExecutionContext = context.dispatcher
+//  private implicit val executionContext: ExecutionContext = context.dispatcher
```
Comment (JustinPihony) [inline src/main/scala/beam/agentsim/agents/modalbehaviors/ChoosesMode.scala:41]: We should remove this altogether

File: src/main/scala/beam/physsim/jdeqsim/PhysSimReplayer.scala [hunk 2]
```diff
@@ -120,7 +120,7 @@ object PhysSimReplayer extends StrictLogging {
           "beam.routing.r5.osmMapdbFile",
           ConfigValueFactory.fromAnyRef(s"""$pwd/r5-simple-no-local/osm.mapdb""")
         )
-        .withValue("beam.exchange.scenario.source", ConfigValueFactory.fromAnyRef(s"""Beam"""))
+        .withValue("beam.exchange.scenario.source", ConfigValueFactory.fromAnyRef("""Beam"""))
```
Comment (JustinPihony) [inline src/main/scala/beam/physsim/jdeqsim/PhysSimReplayer.scala:123]: You can remove the triple quotes also

File: src/main/scala/beam/utils/GpxCompareSimulationVsReference.scala [hunk 3]
```diff
@@ -51,7 +51,7 @@ object GpxCompareSimulationVsReference {
               dstid.foreach {
                 destination =>
                   {
-                    outWriter.write(s"""<rte>""")
+                    outWriter.write("""<rte>""")
```
Comment (JustinPihony) [inline src/main/scala/beam/utils/GpxCompareSimulationVsReference.scala:54]: You can also get rid of the triple quotes.

File: src/main/scala/beam/utils/map/GpxWriter.scala [hunk 4]
```diff
@@ -100,7 +100,7 @@ object GpxWriter {
   }
 
   def drawSourceToDest(outWriter: BufferedWriter, geoUtils: GeoUtils, source: GpxPoint, dest: GpxPoint): Unit = {
-    outWriter.write(s"""<rte>""")
+    outWriter.write("""<rte>""")
```
Comment (JustinPihony) [inline src/main/scala/beam/utils/map/GpxWriter.scala:103]: You can also get rid of the triple quotes.

**Review by JustinPihony — APPROVED** [review]
Minor nits - should be easy to fix if you so want

File: src/main/scala/beam/utils/map/GpxWriter.scala (hunk 4 above)
Comment (carloscaldas) [reply src/main/scala/beam/utils/map/GpxWriter.scala]: done

File: src/main/scala/beam/utils/GpxCompareSimulationVsReference.scala (hunk 3 above)
Comment (carloscaldas) [reply src/main/scala/beam/utils/GpxCompareSimulationVsReference.scala]: done

File: src/main/scala/beam/physsim/jdeqsim/PhysSimReplayer.scala (hunk 2 above)
Comment (carloscaldas) [reply src/main/scala/beam/physsim/jdeqsim/PhysSimReplayer.scala]: done

File: src/main/scala/beam/agentsim/agents/modalbehaviors/ChoosesMode.scala (hunk 1 above)
Comment (carloscaldas) [reply src/main/scala/beam/agentsim/agents/modalbehaviors/ChoosesMode.scala]: done

**Review by rajnikantsh — APPROVED** [review]
LGTM

**Pull request merged**